persistence-model/src/main/resources/application-dev.conf

```diff
@@ -1,18 +1,37 @@
 akka {
   loglevel = "INFO"
-  actor.warn-about-java-serializer-usage = off
+  actor {
+    warn-about-java-serializer-usage = off
+    serializers {
+      institution = "hmda.persistence.serialization.institutions.InstitutionProtobufSerializer"
+    }
+    serialization-bindings {
+      "hmda.persistence.messages.events.institutions.InstitutionEvents$InstitutionCreated" = institution
+      "hmda.persistence.messages.events.institutions.InstitutionEvents$InstitutionModified" = institution
```

Comment:
Testing my knowledge: These two events (`InstitutionCreated` and `InstitutionModified`) are the only ones that need to be listed here because they will be part of the Event Stream. All of the lower-level structures (like `Institution`, `Parent`, `InstitutionType`, etc.) are only needed to serialize the two top-level messages. I'm guessing that's also why `.toBinary` and `.fromBinary` only handle those two message types.

Is that about right? (And if I haven't been clear, let me know; I can try to articulate better)